esnext/package.json

```diff
@@ -31,18 +31,18 @@
   },
   "homepage": "https://github.com/kunalgolani/eslint-config/esnext#readme",
   "dependencies": {
-    "babel-eslint": "~6.1.0",
-    "eslint": "~3.4.0",
-    "eslint-plugin-babel": "~3.3.0",
-    "eslint-plugin-import": "~1.14.0"
+    "babel-eslint": "^8.0.2",
+    "eslint": "^4.0.0",
+    "eslint-plugin-babel": "~4.1.2",
```

Comment:
any particular reason for using `~` here vs `^` everywhere else?

Reply:
`^` is just more flexible because `~` just checks the minor version. As you want 😄 

On the other hand, I should use only one checker (`~` or `^`), I will fix that (next commit)

Reply:
@kunalgolani after thinking (cause of babel package...) `~` is a better choice...

Reply:
A good link to explain the choice for `^` (major version)
http://fredkschott.com/post/2014/02/npm-no-longer-defaults-to-tildes/